build.gradle

```diff
@@ -76,6 +76,13 @@ subprojects {
         delete(file(jarDestinationFolder))
     }
 
+    reportScoverage.doLast {
+        ant.delete dir: "latest-reports"
```

Comment:
in questo modo buildando il sottoprogetto successivo, viene eliminata la build del precedente o sbaglio?